datastructure/graphlist.py

```diff
@@ -0,0 +1,45 @@
+class Vertex:
```

Comment:
its fine to use Vertex  as a name but Node would also work (just mentioning since we have been using node for many other structures)

Reply:
I agree. I used `Vertex `because that was the course use. 
Is it better to change it to `Node`?

Reply:
its just a preference. Vertex does work if its makes better sense for you.